### ISSUE TITLE
Question - How to compile a PTE with dependency on AppSource apps in a build pipeline

### ISSUE THREAD

**Issue by catadumitru** [microsoft/navcontainerhelper#2479]
Scenario:
A PTE app with dependency on 2 AppSource apps where the apps/code are not available for download.
The PTE can be built and deployed with VS Code using symbols.

How to compile, deploy and run tests for this PTE?
Is it possible to download the symbols from an online sandbox and build the app? - are there any helpful commands?

Thanks!

**Comment by freddydk** [comment]
Yes, you can download the symbols from an online sandbox, and you can build the app locally with those, but you cannot deploy it - it won't work as the dependent app is not installed.
Currently the best mechanism you have is to get a runtime package from the app provider and use that.
That can be deployed and used locally for tests.

**Comment by catadumitru** [comment]
If I can get the symbols from an online sandbox, I could build the app and deploy it back on the cloud sandbox.
Maybe it doesn't make much sense to go that way.

With runtime packages I have another issue, we cannot debug even if we set the resource policy.
https://github.com/microsoft/AL/issues/7049 - Cannot see code from runtime packages when "includeSourceInSymbolFile": true

Will try to get a runtime package from the app provider for now.
Thanks for your help!

**Comment by pri-kise** [comment]
@catadumitru you can check out this idea https://experience.dynamics.com/ideas/idea/?ideaid=42616aad-587b-eb11-8fed-0003ff4582f9 and share it, that it'll be priorized

**Comment by catadumitru** [comment]
> @catadumitru you can check out this idea https://experience.dynamics.com/ideas/idea/?ideaid=42616aad-587b-eb11-8fed-0003ff4582f9 and share it, that it'll be priorized

I have voted for it, thanks for sharing.
